### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by officiallyemma** [qwazwsx/infinite-campus#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by officiallyemma** [comment]
test comment

**Reply by dotevie** [reply]
no way! test!

**Comment by sbalbale** [comment]
Is there a way to see when you would have a class using the API?

**Reply by officiallyemma** [reply]
yes, you *should* be able to see that under [Course.placement](https://tonyzimbinski.github.io/infinite-campus/Course.html). It's very possible that they changed the API though. 

BTW: I've been pretty busy but I'll reply to your email later tonight, thank you :-)

**Reply by officiallyemma** [reply]
[heres the relevant code](https://github.com/tonyzimbinski/infinite-campus/blob/master/index.js#L368)

You can see that we actually have to make a request to two separate endpoints, and then combine the grades and placement data. When I wrote it I remember it being very fickle because they didn't provide unique IDs or something like that.
